lib/optimizely/decision_service.rb

```diff
@@ -58,10 +58,14 @@ def get_variation(experiment_key, user_id, attributes = nil)
         return nil
       end
 
-      # Check if user is in a forced variation
-      forced_variation_id = get_forced_variation_id(experiment_key, user_id)
+      # Check if a forced variation is set for the user
```

Comment:
at the start of class, there is a sequence description as comments. Please add forced variation after experiment status.